docs/src/modules/components/ThemeContext.js

```diff
@@ -109,11 +108,8 @@ if (process.env.NODE_ENV !== 'production') {
 
 export function ThemeProvider(props) {
   const { children } = props;
-  const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
-  const pageContextValue = React.useContext(PageContext);
-  // `activePage` does not exist for playground pages
-  // forcing light mode in playground avoids the need for a wrapping theme in playground pages
```

Comment:
I think that a better solution is to move with this direction https://github.com/mui/material-ui/pull/35330/files#diff-41c21e2dc42b8c223fc2fa17580e58041afa590fe34d9fbe9a1ffbe5e17d318b